app/uk/gov/hmrc/ivorchestration/model/core/IvSessionDataCore.scala

```diff
@@ -16,14 +16,26 @@
 
 package uk.gov.hmrc.ivorchestration.model.core
 
-import org.joda.time.DateTime
-import play.api.libs.json.Json
+import play.api.libs.json._
 import uk.gov.hmrc.ivorchestration.model.api.IvSessionData
 
-case class IvSessionDataCore(ivSessionData: IvSessionData, journeyId: JourneyId, createdAt: DateTime)
+import java.time.{Instant, LocalDateTime, ZoneOffset}
+
+case class IvSessionDataCore(ivSessionData: IvSessionData, journeyId: JourneyId, createdAt: LocalDateTime)
```

Comment:
Again, we've switched from Joda to Java time - might be best to leave it as it was, if possible?